en/docs/fhir/guides/fhir-resource-api-template.md

```diff
@@ -0,0 +1,58 @@
+
+Here's an example of how to utilize the Health Tool to generate USCore API templates
+
+1. Start by cloning the example artifacts of particular FHIR profile and extracting them to a preferred location. This includes the ig-uscore/definitions directory, which includes the definition files of the FHIR specification. 
+    * For instance you can find all the latest version of US Core profiles [here](https://www.hl7.org/fhir/us/core/downloads.html#downloadable-copy-of-specification).
+
+2. Navigate to the cloned directory and run the following command to generate the templates,
+
+    ```
+    bal health fhir -m template -o genereated_outputs --org-name healthcare_samples --dependent-package ballerinax/health.fhir.r4.uscore501 ig-uscore/definitions
+    ```
+
+    !!! explanaton 
+
+            * -m [mode] - For this instance mode should be ***template***
+            * -o [output directory] - Location of the generated Ballerina artifacts. If this path is not specified, the output will be written to the same directory from which the command is run.
+            * --org-name - [Organization](https://ballerina.io/learn/package-references/#the-org-field) name of the Ballerina package/template to be generated.
+            * --dependent-package - Fully qualified name of the published Ballerina package containing IG resources (eg: <org\>/<package\>). This option can be used to generate templates specifically for the resources in the given IG. The package name part of this value will be added as a prefix to the template name.
+
+    !!! notes
+            * While genertaing the templates, you can use  ***--included-profile*** or ***--excluded-profile*** to include and exclude whaterver profiles you want. 
+
+    !!! notes
+            You can find all the core packages of different fhir profiles in the [Ballerina central](https://central.ballerina.io/search?q=health.fhir.r4&sort=relevance%2CDESC&page=1&m=packages).
+            
+            For instance:
+
+            * [FHIR base resources](https://central.ballerina.io/ballerinax/health.fhir.r4.international401/latest)
+            * [US Core](https://central.ballerina.io/ballerinax/health.fhir.r4.uscore501/latest)
+            * [AU Base](https://central.ballerina.io/ballerinax/health.fhir.r4.aubase421/latest)
+
+            If you want to generate packages for any custom profiles you can use [Ballerina health package generation tool](https://ballerina.io/learn/health-tool/#package-generation). 
+
+
+3. After you've generated the USCore templates, to run the service as a standalone server, execute the below command.
```

Comment:
Can you mention that these tempaltes doesn't have the business logic implemented, and it should be implemented by the developer

Reply:
Done